src/posit/connect/resources.py

```diff
@@ -61,77 +61,108 @@ def __init__(self, ctx: Context, path: str, pathinfo: str = "", /, **attributes)
         ctx : Context
             The context object containing the session and URL for API interactions.
         path : str
-            The HTTP path component for the collection endpoint
-        pathinfo : str
-            The HTTP part of the path directed at a specific resource
+            The HTTP path component for the resource endpoint
         **attributes : dict
             Resource attributes passed
-
-        Attributes
-        ----------
-        _ctx : Context
-            The context object containing the session and URL for API interactions
-        _path : str
-            The HTTP path for the collection endpoint.
         """
         params = ResourceParameters(ctx.session, ctx.url)
         super().__init__(params, **attributes)
         self._ctx = ctx
-        self._path = posixpath.join(path, pathinfo)
+        self._path = path
 
 
 T = TypeVar("T", bound="Active")
 """A type variable that is bound to the `Active` class"""
 
 
 class ActiveSequence(ABC, Generic[T], Sequence[T]):
-    def __init__(self, ctx: Context, path: str, pathinfo: str = "", uid: str = "guid"):
-        """A sequence abstraction for any HTTP GET endpoint that returns a collection.
+    """A sequence for any HTTP GET endpoint that returns a collection."""
 
-        It lazily fetches data on demand, caches the results, and allows for standard sequence operations like indexing and slicing.
+    _cache: Optional[List[T]]
 
-        Attributes
+    def __init__(self, ctx: Context, path: str, uid: str = "guid"):
+        """A sequence abstraction for any HTTP GET endpoint that returns a collection.
+
+        Parameters
         ----------
-        _ctx : Context
-            The context object containing the session and URL for API interactions
-        _path : str
-            The HTTP path for the collection endpoint.
-        _uid : str
-            The field name used to uniquely identify records.
-        _cache: Optional[List[T]]
+        ctx : Context
+            The context object containing the session and URL for API interactions.
+        path : str
+            The HTTP path component for the collection endpoint
+        uid : str, optional
+            The field name of that uniquely identifiers an instance of T, by default "guid"
         """
         super().__init__()
         self._ctx = ctx
-        self._path = posixpath.join(path, pathinfo)
+        self._path = path
         self._uid = uid
-        self._cache: Optional[List[T]] = None
+        self._cache = None
+
+    @abstractmethod
+    def _create_instance(self, path: str, /, **kwargs: Any) -> T:
+        """Create an instance of 'T'."""
+        raise NotImplementedError()
+
+    def cached(self) -> bool:
```

Comment:
More descriptive name / make private as its more for internal use than subclass use.

```suggestion
    def _has_cache(self) -> bool:
```

Reply:
Making it private is the right call. For some reason, single-word names for things spark joy in me. ✨  

I think the "conventional" name for this is `_is_cached` in most OOP books.